docs/storage/snippets/tutorial/AspireStorage/AspireStorage/AspireStorage/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using AspireStorage.Client.Pages;
+using AspireStorage.Components;
+using Azure.Storage.Blobs;
+using Azure.Storage.Queues;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddAzureBlobService("BlobConnection");
+builder.AddAzureQueueService("QueueConnection");
+
+builder.AddServiceDefaults();
+
+// Add services to the container.
+builder.Services.AddRazorComponents()
+    .AddInteractiveServerComponents()
+    .AddInteractiveWebAssemblyComponents();
+
+var app = builder.Build();
+
+app.MapDefaultEndpoints();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
+{
+    app.UseWebAssemblyDebugging();
+
+    var blobs = app.Services.GetRequiredService<BlobServiceClient>();
+    var docsContainer = blobs.GetBlobContainerClient("fileuploads");
+    await docsContainer.CreateIfNotExistsAsync();
+
+    var queues = app.Services.GetRequiredService<QueueServiceClient>();
+    var queueClient = queues.GetQueueClient("tickets");
+    await queueClient.CreateIfNotExistsAsync();
```

Comment:
Kinda meh. Do this in a hosted service?

Reply:
Happy you brought this up... This is really only a concern when we're doing Azurite, otherwise the tutorial instructs you to create these resources as part of the setup steps. But when using the emulator, what pattern should we suggest? Does it add a lot of complexity to add yet another host service to this solution - and one that doesn't loop, but rather just does one thing? It's kind of similar to needing to seed a database or perform migrations.

Reply:
Another option would be to show how to create the container/queue using Azure Storage Explorer since I think that's the only way to actually view/verify and browse the locally uploaded files, but that adds an additional dependency obviously. I'm not sure how they see their file upload without this though, if they're working with the local emulator a lot they should probably install it anyways.

Reply:
We could add another page (overwriting the counter) for example, and just list all the files uploaded with the support tickets... But then we're making this even more complex. I'm not sure that's what we want either.

Reply:
Why not just make this call in Home.razor's `HandleValidSubmit()` where you get the queue and add something to it? That's the pattern used in the `AspireStorage.Worker` project. It doesn't do this in Program.cs.

Reply:
Isn't that a performance concern? That kind of feels like an antipattern, and that would be really bad in production code, not to mention expensive.

Reply:
Static !?????